src/components/CartInfo/index.tsx

```diff
@@ -0,0 +1,76 @@
+import React, {FC} from 'react';
+import {Text, View} from 'react-native';
+import commonStyles from '../../commonStyles';
+import styles from './styles';
+
+type Props = {
+  data: {
+    item_count: number;
+    display_item_total: number;
+    display_ship_total: string;
+    display_adjustment_total: string;
+    display_tax_total: string;
+    display_total: string;
+  };
+};
+
+const Link: FC<Props> = ({data}) => (
+  <View style={[commonStyles.pane, styles.pane]}>
+    <Text
+      style={[commonStyles.text, commonStyles.textHeader, styles.textHeader]}>
+      Price Details
+    </Text>
+
+    <View style={styles.row}>
+      <Text style={[commonStyles.text, commonStyles.textRegular, styles.text]}>
```

Comment:
To reduce repeating, maybe it will be better to put these style to constant?
```js
const regularTextStyle = [commonStyles.text, commonStyles.textRegular, styles.text];
```